packages/rps/src/redux/store.ts

```diff
@@ -24,28 +24,32 @@ const store = createStore(reducer, enhancers);
 function* rootSaga() {
   yield fork(loginSaga);
 
-  const channelProviderEnabled = yield stateChannelWalletSaga();
-
-  if (channelProviderEnabled) {
-    const client = new RPSChannelClient(new ChannelClient(window.channelProvider));
-    yield fork(channelUpdatedListener, client);
+  let client;
+  if (process.env.AUTO_OPPONENT === 'A' || process.env.AUTO_OPPONENT === 'B') {
+    console.info('Bypassing state channel wallet');
+    client = new RPSChannelClient(new FakeChannelClient('0xOpponent'));
+  } else {
+    client = new RPSChannelClient(new ChannelClient(window.channelProvider));
+    yield stateChannelWalletSaga();
     yield fork(messageQueuedListener, client);
-    yield fork(gameSaga, client);
-
-    if (process.env.AUTO_PLAYER === 'A') {
-      yield fork(autoPlayer, 'A');
-    } else if (process.env.AUTO_PLAYER === 'B') {
-      yield fork(autoPlayer, 'B');
-    }
-
-    if (process.env.AUTO_OPPONENT === 'A') {
-      yield fork(autoOpponent, 'A', client);
-    } else if (process.env.AUTO_OPPONENT === 'B') {
-      yield fork(autoOpponent, 'B', client);
-    } else {
-      yield fork(firebaseInboxListener, client);
-      yield fork(openGameSaga);
-    }
+  }
+
+  yield fork(gameSaga, client);
+  yield fork(channelUpdatedListener, client);
+
+  if (process.env.AUTO_PLAYER === 'A') {
```

Comment:
Nit, might be more elegant as:
``` typescript
if (process.env.AUTO_PLAYER) {
  yield fork(autoPlayer, process.env.AUTO_PLAYER);
}

Reply:
It would be, but that causes a typescript error under the current implementation.